assets/build/install.sh

```diff
@@ -55,6 +55,12 @@ GITLAB_SHELL_VERSION=${GITLAB_SHELL_VERSION:-$(cat ${GITLAB_INSTALL_DIR}/GITLAB_
 GITLAB_WORKHORSE_VERSION=${GITLAB_WORKHOUSE_VERSION:-$(cat ${GITLAB_INSTALL_DIR}/GITLAB_WORKHORSE_VERSION)}
 GITLAB_PAGES_VERSION=${GITLAB_PAGES_VERSION:-$(cat ${GITLAB_INSTALL_DIR}/GITLAB_PAGES_VERSION)}
 
+#download golang
+echo "Downloading Go ${GOLANG_VERSION}..."
+wget -cnv https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-amd64.tar.gz -P ${GITLAB_BUILD_DIR}/
+tar -xf ${GITLAB_BUILD_DIR}/go${GOLANG_VERSION}.linux-amd64.tar.gz -C /tmp/
+ln -s /tmp/go/bin/go /usr/local/bin/go
```

Comment:
using `bin/compile` poses a problem, they use ruby script to run `go install gitlab.com/gitlab-org/gitlab-shell/go/cmd/...`, and when the script does this, it didn't inherit `$PATH` and `$GOROOT` from docker-gitlab's install.sh, and the go executables cannot be found.

So I had to make go available in the default `$PATH`.

@solidnerd